packages/components/src/Banner/Banner.module.css

```diff
@@ -12,10 +12,6 @@
   --svg-icon-size-default: 2.5rem;
 }
 
-.icon {
-  @apply justify-self-center;
-}
-
```

Comment:
This styling doesn't seem to have been doing anything so it wasn't moved over to the new component.